zircon.core/src/commonMain/kotlin/org/hexworks/zircon/api/Fragments.kt

```diff
@@ -9,7 +13,22 @@ import kotlin.jvm.JvmStatic
  */
 object Fragments {
 
+    /**
+     * @see org.hexworks.zircon.api.fragment.MultiSelect
+     */
     @JvmStatic
     fun <M : Any> multiSelect(width: Int, values: List<M>) = MultiSelectBuilder.newBuilder(width, values)
 
+    /**
+     * @see TilesetSelectorBuilder.newBuilder
+     */
+    @JvmStatic
+    fun tilesetSelector(width: Int, vararg componentsToUpdate: TilesetOverride): TilesetSelectorBuilder = TilesetSelectorBuilder.newBuilder(width, *componentsToUpdate)
```

Comment:
We have `Group` which does the same as `componentsToUpdate`. You can take a look at how it works in `GroupExample`. You can use a `Group` to have a collection of `Component`s which are bound together in terms of themes, tilesets and more. I think we should have the option to use a `Group` instead of `componentsToUpdate` and we can use a `Group` internally.